### PR TITLE
Implement more wnd callbacks (single player menu, exit in-game)

### PR DESCRIPTION
**Pull request by tgjones**

### CODE REVIEW

File: src/OpenSage.Game/Content/ModelLoader.cs [hunk 1]
```diff
@@ -105,7 +105,7 @@ protected override Model LoadEntry(FileSystemEntry entry, ContentManager content
                     subObjects.Add(new ModelSubObject(w3dSubObject.Name, bone, mesh));
                 }
             }
-            else
+            else if (w3dMeshes.Count > 0)
```
Comment (feliwir) [inline src/OpenSage.Game/Content/ModelLoader.cs]: I guess this fixes a crash?

File: src/OpenSage.Game/Game.cs [hunk 2]
```diff
@@ -423,11 +435,56 @@ public void StartGame(string mapFileName, IConnection connection, PlayerSetting[
...
+                0,
+                false);
+        }
+
+        public void StartMultiPlayerGame(
+            string mapFileName,
+            IConnection connection,
+            PlayerSetting[] playerSettings,
+            int localPlayerIndex)
+        {
```
Comment (feliwir) [inline src/OpenSage.Game/Game.cs]: This will change in the future?

File: src/OpenSage.Game/Content/ModelLoader.cs (hunk 1 above)
Comment (tgjones) [reply src/OpenSage.Game/Content/ModelLoader.cs]: Yes, the one you found in CHI01. Now that we can load it from the menu, it seemed somewhat related :)

File: src/OpenSage.Game/Game.cs (hunk 2 above)
Comment (tgjones) [reply src/OpenSage.Game/Game.cs]: It will need a lot more code than what's there now, if that's what you mean?

**Pull request merged**